extra_geom/tests/test_geometry_old.py

```diff
@@ -16,7 +16,7 @@
 
 
 def test_inspect():
-    with h5py.File(pjoin(tests_dir, 'lpd_mar_18.h5')) as f:
```

Comment:
This is unrelated, but it was producing warnings in the test, and it's an easy fix.